src/main/webapp/dataverse.xhtml

```diff
@@ -451,8 +451,7 @@
                                                     <li class="#{SearchIncludeFragment.mode == 'search' ? '' : 'disabled'}">
                                                         <p:commandLink action="#{DataversePage.setupLinkingPopup('savedSearch')}"
                                                                        oncomplete="PF('linkDataverseForm').show()"
-                                                                       update=":linkDataverseForm"
-                                                                       disabled="#{SearchIncludeFragment.mode != 'search'}">
```

Comment:
why remove this?

Reply:
It's handled in the class statement at 451.

Reply:
Ah. True, both don't seem to be needed. I wonder if one way is better than the other?

Reply:
Mike might have a preference. I was just trying to simplify.